File: Freedom/微能量/HOME/EnergyDetailViewController.m
//  EnergyDetailViewController.m
//  Created by Super on 16/9/5.
//  Copyright © 2016年 Super. All rights reserved.
//
#import "EnergyDetailViewController.h"
@implementation EnergyDetailViewController{
    UIWebView *web;
}
-(void)viewDidLoad{
    [super viewDidLoad];
    NSURL *url = [NSURL URLWithString:@"http://weibo.com/tv/v/ErlcFmAdZ?from=vhot"];
    NSURLRequest *request = [NSURLRequest requestWithURL:url];
    web = [[UIWebView alloc]initWithFrame:self.view.bounds];
    [web loadRequest:request];
    [self.view addSubview:web];
}
@end

File: Freedom/微能量/HOME/EnergyDetailViewController.swift
//
//  EnergyDetailViewController.swift
//  Freedom
import UIKit
class EnergyDetailViewController: EnergyBaseViewController{
    override func viewDidLoad() {
        super.viewDidLoad()
        let web = UIWebView(frame:view.bounds)
        let url = URL(string:"http://weibo.com/tv/v/ErlcFmAdZ?from=vhot")
        let request = URLRequest(url: url!)
        web.loadRequest(request)
        view.addSubview(web)
    }
}
